ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -11965,6 +11965,53 @@ def test_torch_tensor_quantile(
     )
 
 
+# random_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="random_",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=["float16", "float32", "float64", "int32", "int64"],
```

Comment:
why are you restricting the data types? Any particular reason?

Reply:
having `int8` with other numerical datatypes not supported with this error message I received in all backends.

> ValueError: Argument `dtype` got invalid value <dtype: 'complex64'>. Accepted dtypes are (tf.float16, tf.bfloat16, tf.float32, tf.float64, tf.int32, tf.int64).

Reply:
but still you're not including the bfloat